code/modules/mob/living/carbon/xenomorph/Xenomorph.dm

```diff
@@ -269,6 +269,8 @@
 	var/icon_xeno
 	var/icon_xenonid
 
+	var/screech_status_flags = 0
```

Comment:
as i understand it this flag is used as a single variable that tells you if the xeno is buffed by a royal rn or not. this would definitely be better off as a variable that holds bitflags that indicate various buffs or not the xeno might have for the future

such as 

`var/xeno_effect_flags = ROYAL_ROAR_ENHANCED`
and checked with
`if(X.xeno_effect_flags & ROYAL_ROAR_ENHANCED)`

this would also let it be used for other possible buffs either by refactoring preexisting buffs or the shitty pheromone variables to also be bitflags, also helpful for future castes or strains

`#define ROYAL_ROAR_ENHANCED (1<<1)`